plugins/action/zos_copy.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) IBM Corporation 2019, 2020, 2021, 2022
+# Copyright (c) IBM Corporation 2019-2023
```

Comment:
I don't think this is a biggie, but it is inconsistent... other changes have year, year, year, etc.  This has year-year.  Minor nit-pick

Reply:
@richp405 - I have not followed up with legal but the reason i compressed it down from the comma delimited is because a range can be used for sequential yrs.

My understanding is a range in  software programs that are constantly being revised and updated incrementally can be used like in the change I pushed. The ‘start’ year should be the date of the oldest content still contained in the work, with the ‘end’ year being the last year that the work was revised.

What I am looking into is what if we skip a year after a range, where with commas its easy to identify the year of the change but will use up space, once we have entered into a hyphenated model, we only focus on the start and end dates and not in between.